compiler/fir/analysis-tests/testData/resolve/extendedCheckers/SafeInitializationChecker/fullConstructionAnalysis3.kt

```diff
@@ -0,0 +1,19 @@
+// WITH_STDLIB
+
+open class E {
+    open val x = "Hello"
+    open fun foo(bool: Boolean) = x
+}
+
+interface I {
+    fun foo(bool: Boolean) = "Hello"
+}
+
+class F : E(), I {
+    val y = foo(true)
+    <!ACCESS_TO_UNINITIALIZED_VALUE, ACCESS_TO_UNINITIALIZED_VALUE!>override val x = foo(false)<!>
```

Comment:
Why do we have two diagnostics here?

Reply:
Because we attach a warning to the property where the initialization error occurred and create a trace from the place where the initialization error occurred (in the figure(Of course this isn't a version of warnings for users)).

We also faced the problem that only one of these warnings is displayed, but we didn't have time to deal with this problem.

![image](https://user-images.githubusercontent.com/43092363/174411653-bcac676a-f902-4ebd-9fcb-27a435600083.png)